### ISSUE TITLE
Respecting formating when importing text data

### ISSUE THREAD

**Issue by mroyce1** [UniversalDataTool/universal-data-tool#378]
Hi, 

I was wondering whether there is a way to preserve the text formating when importing text data. I'm currently working on a NER problem where it would be nice if the structure of the text could be preserved. For example, `'\n'` is simply displayed as `'\ n'` in UDT, even though I'd like it to be interpreted as a line break. Is there a way to change that?

Thanks!

**Comment by seveibar** [comment]
Yes with `wordSplitRegex`! More info on the [format page](https://github.com/UniversalDataTool/udt-format/blob/master/interfaces/text_entity_relations.md)

I think to fix it you'll want to set "wordSplitRegex" to `[a-zA-ZÀ-ÿ\\]+`

Related to #374
